lib/turnsole/heliotrope/service.rb

```diff
@@ -272,13 +280,21 @@ def product_institution_subscribed?(product_identifier:, institution_identifier:
         response.success?
       end
 
-      def subscribe_product_institution(product_identifier:, institution_identifier:)
+      def subscribe_product_institution(product_identifier:, institution_identifier:, license: :full)
```

Comment:
Is the idea to delete this method (as well as `subscribe_product_institution`) once Greensub switches to  using `set_product_institution_license`? If so please note this in HELIO-3616.

Reply:
Also, the specs would need to be to be update to use the new setter method.

Reply:
I'm not totally sure. I think @gkostin1966 should probably chime in here.

Reply:
The original idea was to be backward compatible.  Since we are no longer concerned with being backward compatible it would be best to purge all the subscribe methods which will probably make changing Greensub easier because it will blow up if it references a subscribe method.

Reply:
Got it. I'll make the changes.